cmd/ocm-backplane/login/login.go

```diff
@@ -121,16 +121,56 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 		"ID":   clusterID,
 		"Name": clusterName}).Infoln("Target cluster")
 
+	// validate kubeconfig save path when login into multi clusters
+	if args.kubeConfigPath != "" {
+		if !args.multiCluster {
```

Comment:
Looks the current test doesn't cover the logic here based on the generated coverage info:
![login_test_coverage](https://github.com/openshift/backplane-cli/assets/5784943/bf1e190f-ca26-4cfb-89b5-8a47ffda444c)

Reply:
I think that's true, but I did not touch this block... I just moved it to several lines above.

Reply:
tried to implement part of this as well